site/source/user-manual/service-guides/vaultwarden/bitwarden-walkthrough.rst

```diff
@@ -0,0 +1,457 @@
+            
+        .. warning:: Do not lose it - you will not be able to log in without it!
+
+    #. Enter a name for the account, a strong password and (optionally) a password hint - then click **Create account**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-account-creation.png
+                    :width: 50%
+                    :alt: vaultwarden-account-creation
+    
+        .. warning:: It is important to realize that this is the 'Master password' for all your other passwords. Make it very strong, memorize it, write it down, and back it up to a safe place. If you lose it, you may lose access to all your passwords and your entire digital life.
+
+    #. Now you can log in to your new password manager! Enter your email to login:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-login-email.png
+                    :width: 50%
+                    :alt: vaultwarden-login-email
+
+    #. Then your master password:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-login-password.png
+                    :width: 50%
+                    :alt: vaultwarden-login-password
+
+    #. We are greeted with our newly setup Vaultwarden password vault!
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-fresh-vault.png
+                    :width: 50%
+                    :alt: vaultwarden-fresh-vault
+    
+    #. At the top you can visit **Tools** for reports and to use the password generator. You can also import data from another program, such as lastpass, onepass, or keepass!
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-tools.png
+                    :width: 50%
+                    :alt: vaultwarden-tools
+    
+    #. **Account settings** gives you all your options, including the ability to set up 2 factor authorization.
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-account-settings.png
+                    :width: 50%
+                    :alt: vaultwarden-account-settings
+
+    #. Now you're ready to start storing some passwords!
+
+.. collapse:: Storing your first password
+
+    #. Back at the main page for your vault, let's do a quick example login. We'll start by creating a folder for it:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-click-add-folder.png
+                    :width: 50%
+                    :alt: vaultwarden-click-add-folder
+    
+    #. Name the folder and click **Save**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-add-folder.png
+                    :width: 50%
+                    :alt: vaultwarden-add-folder
+
+    #. Click **Add item**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-add-item.png
+                    :width: 50%
+                    :alt: vaultwarden-add-item
+
+    #. Now add the credentials and click **Save**:
```

Comment:
Since you are making a detailed guide, this would be a good place to mention that an entry can have multiple URLs, and why that is especially beneficial when used with our OS - you can give an example .com, .local, .onion, .i2p, a subdomain, etc - this can also tease the soon-to-come dynamic interfaces and clearnet